compiler/rustc_target/Cargo.toml

```
@@ -12,7 +12,9 @@ rustc_fs_util = { path = "../rustc_fs_util" }
 rustc_macros = { path = "../rustc_macros" }
 rustc_serialize = { path = "../rustc_serialize" }
 rustc_span = { path = "../rustc_span" }
+serde = { version = "1.0.219", features = ["derive"] }
```

using serde with the derive feature causes a hard dependency on syn/procmacro2 that leads to everything that uses serde but doesn't use derive needing to wait on these expensive crates to build
The current suggested way to pull serde derive (and increase build parallelization) is to just pull the derive crate separately and use serde_derive::Deseralize etc in code
```suggestion
serde = "1.0.219"
serde_derive = "1.0.219"
```

---

nice suggestion, yes please do this